tda/config.local.yaml

```diff
@@ -1,3 +1,5 @@
+collect_mock_data: true
+
```

Comment:
@asottile-sentry so `dsn` defined on line 7 in this file is the DSN for reporting TDA's own crashes (`empower-tda` project in demo.sentry.io)
Remind me please why are you setting it to mini-relay on localhost? We shouldn't nee to capture those, they are for debugging TDA itself, right?